<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popup_Histogram</class>
 <widget class="QDialog" name="popup_Histogram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Measurements</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>620</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Features">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>361</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>Features</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="7" column="1">
     <widget class="QCheckBox" name="checkBox_Range">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Range</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QCheckBox" name="checkBox_CoeffOfVar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>굴림</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Coefficient of Variation</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_Mean">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Mean</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBox_Kurtosis">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Kurtosis</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBox_Variance">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Variance</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QCheckBox" name="checkBox_RobustMeanAbsDev">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>굴림</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Robust Mean Abs Dev</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QCheckBox" name="checkBox_MaxHisGradient">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Max Hist Gradient</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QCheckBox" name="checkBox_MinHistGradGrey">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Min Hist Gradient Grey</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="checkBox_90thPercentile">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>90th Percentile</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="checkBox_Minimum">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Minimum</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBox_Maximum">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Maximum</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="checkBox_InterquartileRange">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Interquartile range</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="checkBox_Mode">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Mode</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBox_10thPercentile">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>10th Percentile</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_Skewness">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Skewness</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBox_Median">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Median</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="checkBox_MedianAbsDev">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>굴림</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Median Absolute Deviation</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QCheckBox" name="checkBox_QuartileCoeff">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Quartile Coefficient</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QCheckBox" name="checkBox_Entropy">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Entropy</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QCheckBox" name="checkBox_Uniformity">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Uniformity</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QCheckBox" name="checkBox_MaxHistGradGrey">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Max Hist Gradient Grey</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QCheckBox" name="checkBox_MinHistGradient">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Min Hist Gradient</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="checkBox_MeanAbsDev">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>굴림</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Mean Absolute Deviation</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_nBins">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Number of Bins</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_nBins">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentText">
       <string>8</string>
      </property>
      <property name="maxVisibleItems">
       <number>5</number>
      </property>
      <property name="maxCount">
       <number>64</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>64</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>popup_Histogram</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>popup_Histogram</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
